### PR TITLE
R cmd check fixes

### PR DESCRIPTION
**Pull request by GaryTruong**
R cmd check is now passing

### CODE REVIEW

**Comment by GaryTruong** [comment]
@dfalster Should we merge this in and then create a new branch to work through the build errors? Or just keep working on this branch?

File: DESCRIPTION [hunk 1]
```diff
@@ -58,6 +57,7 @@ Imports:
 Suggests:
     austraits,
     bench,
+    data.table,
```
Comment (dfalster) [inline DESCRIPTION:60]: Can we please remove datat.table. Find whatever function uses it and an alternative from existing packages. It's quite a heavy decency and I'd rather not include it.

**Review by dfalster — APPROVED** [review]

**Comment by dfalster** [comment]
Sure, happy to merge

**Comment by GaryTruong** [comment]
The test build examples and test constancy are passing now. Build Austraits is failing at NSWFRD_2014 with that `str` error

**Pull request merged**